src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/XABuildPaths.cs

```diff
@@ -39,8 +40,8 @@ static string GetTestDirectoryRoot ()
 			if (Directory.Exists (_testOutputDirectory))
 				return _testOutputDirectory;
 
-			// Set when running on Azure Pipelines https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables
-			var rootDir = Environment.GetEnvironmentVariable ("BUILD_STAGINGDIRECTORY");
+			// Set when running on Azure Pipelines https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables or when part of the megapipeline
+			var rootDir = Environment.GetEnvironmentVariable ("MEGAPIPELINE_ANDROID_DIR") ?? Environment.GetEnvironmentVariable ("BUILD_STAGINGDIRECTORY");
```

Comment:
`BUILD_STAGINGDIRECTORY` should be set in all pipelines, do we actually need to change this test output directory?

Reply:
That's a mistake from when I was debugging. No need.